### ISSUE TITLE
GSSOC'2023 I will start assigning the issues from 22nd May 2023

### ISSUE THREAD

**Issue by akshitagupta15june** [akshitagupta15june/PetMe#466]
Join discord channel of Moksh for any queries: https://discord.com/invite/Jmc97prqjb

**Comment by oputaolivia** [comment]
Hello I am Olivia a contributor from GSSoC, I want to know if it’s possible to bring in new features aside from improving current features.

**Comment by rathi710** [comment]
@oputaolivia yes its possible , you can raise an issue regarding the new feature that you want to improve

**Comment by oputaolivia** [comment]
Thank you
I already raised one issue concerning user Authentication

**Comment by Hritik1503** [comment]
completed

**Comment by ajaymahiwal** [comment]
Hello to Maintains and Everyone Who is reading,
Can you tell me what kind of new features you want in this project or any updates ???
Which things should i update ??

**Comment by pavlosts** [comment]
Hey everyone! I am a UI/UX designer looking to contribute to my first open-source project. Is there anything I could do to help?

Also, I can't accept the Discord invite for some reason
